spec/netbox_client_ruby/connection_spec.rb

```diff
@@ -45,8 +45,13 @@
     end
 
     it 'sets the adapter' do
-      expect(NetboxClientRuby::Connection.new.builder.handlers)
-        .to include Faraday::Adapter::NetHttp
+      if Faraday::VERSION < '1.0'
+        expect(NetboxClientRuby::Connection.new.builder.handlers)
+          .to include Faraday::Adapter::NetHttp
+      else
+      expect(NetboxClientRuby::Connection.new.builder.adapter)
```

Comment:
This seems to not be indented correctly?

Reply:
Fixed